### DISCUSSION TITLE
How can I set up REST dataSource using URL with dynamic key?

### DISCUSSION THREAD

**Discussion by GarryXian** [Budibase/budibase#3230] [Help]
I am using APIM (API management) product provided by Azure to expose API for master data. A APIM url need to include a dynamic secret created by apim key and timestamp using sha256.
So i wonder if anyone has met the similar scenros and how you relove?

**Comment by mjashanks** [comment]
Hi @GarryXian.

Something similar is discussed here: https://github.com/Budibase/budibase/discussions/2685

We are planning a big update to the REST connector in December 2021. As this is a common scenario, we would hope to cover it properly then.

**Reply by GarryXian** [reply]
hi Thx for your replay, may i know the exact date when will the update be released?

**Reply by mjashanks** [reply]
We usually push out updates at the end of every month. Since December is a bit of a wind-down period for us, I'd expect it to come in the first week of January 2022.
